pkg/common/migrations.go

```diff
@@ -429,18 +435,23 @@ func MigrateFromPipe(log types.Logger, met metrics.SiloMetrics, instanceID strin
 }
 
 type MigrateToOptions struct {
-	Concurrency int
-	Compression bool
+	Concurrency     int
+	Compression     bool
+	CompressionType packets.CompressionType
 }
 
 /**
  * Migrate TO a pipe
  *
  */
-func MigrateToPipe(ctx context.Context, readers []io.Reader, writers []io.Writer,
+func MigrateToPipe(log types.Logger, ctx context.Context, readers []io.Reader, writers []io.Writer,
```

Comment:
Minor nit-picking, but the usual convention is to always have `ctx` as the first parameter:
> The Context should be the first parameter, typically named ctx

https://pkg.go.dev/context